toptal/haste-server
# Convert haste-server to typescript

### Description

Converted haste server to typescript, added express.js server. 

- Converted to typescript
- Converted server to express
- Restructure the code
- Change package manager to yarn
- Added config folder and moved config files to here.

Next:
- Add unit tests for stores
- Update Readme
- Convert callbacks to async functions
- Improve eslint and tsconfig configuration files

These developments are done according to monorepo-approach server example project.

### How to test

- `yarn` 
- `yarn dev`
- check if server works as expected

second way:
- docker-compose up
- check if server works as expected
